src/services/API/Endpoints.ts

```diff
@@ -1,3 +1,4 @@
 export const quizEndpoint = "/quiz/";
 export const sessionsEndpoint = "/sessions/";
 export const sessionAnswersEndpoint = "/session_answers/";
+export const externalAuthTokenEndpoint = "/check-auth-token/"
```

Comment:
why not simply?

+ if we are changing the endpoint in the backend based on my comments, we'll have to change that here too.

```suggestion
export const authenticationEndpoint = "/check-auth-token/"
```